spec/rubocop/rspec/description_extractor_spec.rb

```diff
@@ -42,15 +42,24 @@ def bar
     YARD::Registry.all
   end
 
+  let(:temp_class) do
+    temp = RuboCop::Cop::Cop.dup
+    temp.class_exec do
+      class << self
+        undef inherited
+        def inherited(*) end
+      end
+    end
+    temp
+  end
+
   def stub_cop_const(name)
-    stub_const(
-      "RuboCop::Cop::RSpec::#{name}",
-      Class.new(RuboCop::Cop.const_get(:WorkaroundCop))
-    )
+    stub_const("RuboCop::Cop::RSpec::#{name}", Class.new(temp_class))
   end
 
   before do
     stub_cop_const('Foo')
+    stub_const('RuboCop::Cop::RSpec::Foo::HelperClassForFoo', Class.new)
```

Comment:
Why is this necessary now?

Reply:
Nice catch! Something was breaking in the process, but doesn't now.
Removing.